lib/portlayer/exec/commit.go

```diff
@@ -138,6 +138,11 @@ func Commit(ctx context.Context, sess *session.Session, h *Handle, waitTime *int
 			if s.ExtraConfig != nil && h.TargetState() == StateStopped && h.Runtime.PowerState != types.VirtualMachinePowerStatePoweredOff {
 				detail := fmt.Sprintf("Reconfigure: collision of concurrent operations - expected power state poweredOff, found %s", h.Runtime.PowerState)
 				log.Warn(detail)
+
+				// log out current vm power state and runtime power state got from refresh, to see if there is anything mismatch,
+				// cause in issue #6127, we see the runtime power state is not updated even after 1 minute
+				ps, _ := h.vm.PowerState(ctx)
+				log.Debugf("Container %s power state: %s, runtime power state", h.ExecConfig.ID, ps, h.Runtime.PowerState)
```

Comment:
you have 3 arguments but 2 formatting verbs here

Reply:
oh.. no